pynest/nest/tests/test_sp/test_all.py

```diff
@@ -60,9 +62,10 @@ def suite():
     test_suite.addTest(test_growth_curves.suite())
     test_suite.addTest(test_sp_manager.suite())
     test_suite.addTest(test_synaptic_elements.suite())
-    test_suite.addTest(test_disconnect.suite())
+    #test_suite.addTest(test_disconnect.suite())
```

Comment:
Why are you commenting out the disconnection test? Either activate it again or delete the line entirely.